Java/Java.sublime-syntax

```diff
@@ -1128,8 +1128,19 @@ contexts:
     - match: \n
       scope: invalid.illegal.newline.java
       pop: true
-    - match: \\.
+    - include: strings-escape
+
+  strings-escape:
```

Comment:
Many contexts normally use plural (see: `keywords`, `constants-and-special-vars`, ...). So the context should propably named `string-escapes`.

Reply:
I've matched it with `strings-common` but agree it's better to use plurals like everywhere else.